docs/_tutorials/datasets/dataset.rst

```diff
@@ -36,6 +36,8 @@ Existing Datasets
 +----------------------+------------+------------+--------------------+--------+
 | **CORe50-v2-391**    | 50         | 224x224x3  | YES                | Images |
 +----------------------+------------+------------+--------------------+--------+
+| **Stream-51**        | 51         | ?x?x3      | YES                | Images |
```

Comment:
Est ce qu'il faut pas en plus faire une partie dans scenarios suite? ya plusieur scenarios possible dans Stream-51 non?

Reply:
Yes I though about putting in another PR

Reply:
See #100 , if you want to can do it :p